package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ovirt-ui-components",
-  "version": "0.2.6",
```

Comment:
There is no need for this, I'll increase that automatically when publishing the package

Reply:
Ok, I'll back that change out.